apiserver/pom.xml

```diff
@@ -735,6 +702,7 @@
                             <mainClass>alpine.embedded.EmbeddedJettyServer</mainClass>
                         </manifest>
                     </archive>
+                    <attachClasses>true</attachClasses>
```

Comment:
[Necessary](https://maven.apache.org/plugins/maven-war-plugin/war-mojo.html#attachClasses) for the `apiserver` to be usable as dependency in other modules (i.e. `coverage-report`). Normally, modules with `packaging=war`can't be used as dependency.